include/wsrep/server_state.hpp

```diff
@@ -180,8 +181,10 @@ namespace wsrep
 
         virtual ~server_state();
 
-
+        wsrep::encryption_service* encryption_service()
+        { return encryption_service_; }
```

Comment:
Why we need a public access to `encryption_service`? It should be used only internally by `server_state`. Is is because of encryption key getters and setters are part of the service interface?

Reply:
Encrypt_cb needs to use do_crypt function.